theme/nuxt.config.js

```diff
@@ -1,11 +1,6 @@
 import webpack from 'webpack';
 
 export default {
-  mode: 'universal',
```

Comment:
This was removed since it's shouldn't be specified here, it's default configs and specifying them here will block normal Nuxt behavior in regards to customizations via .env file